operation/fhir-operation-bulkdata/pom.xml

```diff
@@ -48,5 +48,13 @@
             <artifactId>jsonassert</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
not really related to this PR, but can we flip the `jakarta.ws.rs-api` above to provided?